Essentials/src/plugin.yml

```diff
@@ -7,6 +7,7 @@ website: http://tiny.cc/EssentialsCommands
 description: Provides an essential, core set of commands for Bukkit.
 softdepend: [Vault, LuckPerms]
 authors: [Zenexer, ementalo, Aelux, Brettflan, KimKandor, snowleo, ceulemans, Xeology, KHobbits, md_5, Iaccidentally, drtshock, vemacs, SupaHam, md678685]
+contributors: [LaxWasHere, pop4959, JRoy, Triagonal, zml2008]
```

Comment:
```suggestion
authors: [md_5]
contributors: [does, not, matter ]
```

Reply:
Wow that's controversial

Reply:
Added 2008Chooco to the contributors list for making the contributors list

Reply:
`2008Choco` is not a contributor to EssentialsX

Reply:
he invited contributors

Reply:
no u